src/extension/extension.ts

```diff
@@ -47,14 +47,13 @@ import { getDefaultBazelExecutablePath } from "./configuration";
  *
  * @param context The extension context.
  */
-export function activate(context: vscode.ExtensionContext) {
+export async function activate(context: vscode.ExtensionContext) {
   const workspaceTreeProvider = new BazelWorkspaceTreeProvider(context);
   const codeLensProvider = new BazelBuildCodeLensProvider(context);
   const buildifierDiagnostics = new BuildifierDiagnosticsManager();
   const completionItemProvider = new BazelCompletionItemProvider();
 
-  // tslint:disable-next-line:no-floating-promises
-  completionItemProvider.refresh();
+  void completionItemProvider.refresh();
```

Comment:
For the no-floating-promises violations, instead of disabling those with `void`, can you use `eslint-disable-next-line`? I think these should be audited at some point as they could constitute bugs, and `eslint-disable-next-line` will be easier to audit than `void`.

Reply:
Makes sense, I will fix in the coming days.